webxrlayers-1.bs

```diff
@@ -348,6 +349,11 @@ enable optical chromatic aberration correction for the layer. If the user agent
 should return <code>null</code> on getting, and setting should be a <code>no-op</code>.
 If the {{XRCompositionLayer/chromaticAberrationCorrection}} was changed, it will take effect at the next {{XRFrame}}.
 
+The <dfn attribute for="XRCompositionLayer">forceMonoPresentation</dfn> attribute forces the right eye to use the same layer configuration
+as the left eye. This MAY be used to remove the {{XRLayerLayout/stereo}} effect of a {{XRCompositionLayer}}. Setting this to <code>true</code> on a {{XRLayerLayout/mono}} layer has no effect.
```

Comment:
nit: shoudl it throw a TypeError instead?

Reply:
No. We discussed during tpac and agreed it should be ignored